docker_entrypoint.sh

```diff
@@ -38,8 +38,8 @@ sed -i 's/\(BEGIN\|END\) PRIVATE KEY/\1 EC PRIVATE KEY/g' /root/.lnd/tls.key
 
 configurator
 configurator_child=$!
-if test -f /root/.lnd/requires.reset-txs; then
-  rm /root/.lnd/requires.reset-txs &
+if [ -e /root/.lnd/requires.reset_txs ]; then
```

Comment:
-f would have been fine "file exists and is a regular file."
-e = "file exists"
-w might even be most proper since if the file isn't writable it's not deletable and you're `rm`ing right afterward.
But it truly doesn't matter which you select because docker_entrypoint.sh is presumably run as root.